packages/grafana-data/src/utils/unaryOperators.ts

```diff
@@ -47,5 +48,11 @@ export const unaryOperators = new Registry<UnaryOperatorInfo>(() => {
       operation: (value: number) => Math.ceil(value),
       unaryOperationID: UnaryOperationID.Ceil,
     },
+    {
+      id: UnaryOperationID.Percent,
+      name: 'Percentage',
```

Comment:
I am not 100% sure we want to call this `percentage` (unless we do 0-100) ), would `normalize` fit better? The issue there is that it may be confused with vector normalization.